src/Compilers/Core/Portable/RuleSet/RuleSetInclude.cs

```diff
@@ -78,11 +83,9 @@ public RuleSet LoadRuleSet(RuleSet parent)
         private string GetIncludePath(RuleSet parent)
```

Comment:
Is there even a reason to have a separate function here? There appears to be only a single caller.